advancedTodo/src/redux/todoAppComplete/todoApp.ui.js

```diff
@@ -61,8 +62,8 @@ class TodoApp extends Component {
 	}
 

```

Comment:
The variable `value` in `handleOnChange` can be declared with `const` as it is not reassigned.